recipes/libbsc/meta.yaml

```diff
@@ -0,0 +1,41 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+{% set name = "libbsc" %}
+{% set version = "3.1.0a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/shubhamchandak94/libbsc/archive/v{{ version }}.tar.gz
```

Comment:
Can you use upstream source instead of your fork?

Reply:
I actually made a few minor fixes related to multithreading in my fork, that's why I've been sticking to it. I am happy to provide appropriate credit to the original author if you think that's needed.

Reply:
Can you use upstream source and add your changes as a patch file so that anybody can see what changed?

Reply:
I can do that. Should the patch be put in the recipe folder?

Reply:
See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#patches

Reply:
I am working on this and have one more question. The last release on the upstream github repo (https://github.com/IlyaGrebnov/libbsc) is quite old. There have been 8 reasonably large commits on the upstream repo and 3 on my fork after that. Is it okay to include all these in a patch file (size 686KB)?

Reply:
Pushed it now.

Reply:
Have you talked with upstream authors about making a release?

Reply:
I don't know the person and I know someone who contacted them (for some other reason) and didn't get a response. If you think that's required, I can create an issue there.

Reply:
Yes, please contact upstream about a release.